selenium-system-test/src/test/resources/features/checkEligibility/Applicant goes through eligibility check.feature

```diff
@@ -1,27 +1,42 @@
 @HTS-90
 Feature: Applicant goes through eligibility check
 
-  Scenario: User is in receipt of working tax credit and so is eligible
-    Given a user is in receipt of working tax credit
+  Scenario: User is in receipt of WTC and so is eligible
+    Given a user is in receipt of WTC
     When they apply for Help to Save
     Then they see that they are eligible for Help to Save
 
   @BrowserStack
```

Comment:
just curious, why only NON eligible scenerios have `@BrowserStack` annotation ?

Reply:
i guess, the annotation is applicable to all scenerios in the file ? check with Marcus if its required at all and if needed it  should be added at the top